internal/exp/metrics/metadata.yaml

```diff
@@ -1,4 +1,4 @@
 status:
   disable_codecov_badge: true
   codeowners:
-    active: [RichieSams, tombrk]
```

Comment:
Instead of removing, you can move yourself to `emeritus` if you want. Same for other files